packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteView/__tests__/useDeleteView.spec.tsx

```diff
@@ -257,13 +257,13 @@ describe('useDeleteView', () => {
     expect(result.current).toEqual(
       expect.objectContaining({
         disableCancel: true,
-        disableClose: true,
-        disablePrimary: true,
+        disableClose: false,
```

Comment:
@reesscot this behavior change is acceptable? Basically close is now enabled when everything is complete